mama_ng_contentstore.env

```diff
@@ -0,0 +1,10 @@
+# Add Environment Variables
+
+
+SECRET_KEY=6(16ds+i2+%ik6z&!yer+ga9m=e%jcqiz_5wszg)r-z!2--b1d
```

Comment:
Yeah - if you end up using docker-compose on the server I think these .env files would be managed by Puppet like local_settings.py currently - this setup is an attempt at a sane dev env at the moment